backend/api/application_form/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+from django.core.validators import RegexValidator
+
+# Create your models here.
+class ApplicationForm(models.Model):
```

Comment:
Kind of a nit, but maybe something like ApplicantInfo would be more similar to what we're actually returning when we hit the endpoint. Can we rename this if it's not too much work?
https://stackoverflow.com/questions/25091130/django-migration-strategy-for-renaming-a-model-and-relationship-fields

Reply:
I don't have much of a preference, but perhaps we should go with Doug's suggestion